packages/test-utils/src/reporter.ts

```diff
@@ -900,12 +904,18 @@ class ReplayReporter {
       output.push(`\n🚀 Successfully uploaded ${uploads.length} recordings:\n`);
       const sortedUploads = sortRecordingsByResult(uploads);
       sortedUploads.forEach(r => {
-        output.push(
-          `   ${getTestResultEmoji(r)} ${(r.metadata.title as string | undefined) || "Unknown"}`
-        );
-        output.push(
-          `      ${process.env.REPLAY_VIEW_HOST || "https://app.replay.io"}/recording/${r.id}\n`
-        );
+        if (r.status === "uploaded" || r.status === "crashUploaded") {
+          output.push(
+            `   ${getTestResultEmoji(r)} ${(r.metadata.title as string | undefined) || "Unknown"}`
+          );
+          if (r.status === "uploaded") {
+            output.push(
+              `      ${process.env.REPLAY_VIEW_HOST || "https://app.replay.io"}/recording/${r.id}\n`
+            );
+          } else {
+            output.push(`      Browser crashed while recording. Report uploaded to replay.\n`);
```

Comment:
"Report uploaded to replay." sounds a little ambiguous, like the user can access the report. If we can something like "Replay team have been notified", or something along the line because we are not really alerted for each crash.